src/components/calculatorCards/bmrCalculator.tsx

```diff
@@ -0,0 +1,164 @@
+import React, { useState } from "react";
+import { Button, Card, Col, Form, Row, DropdownButton, Dropdown } from "react-bootstrap";
+import "./calculatorStyles.css"
+
+let age: number = 0;
+let height: number = 0;
+let weight: number = 0;
+
+ // BMR (kcal / day) = 10 * weight (kg) + 6.25 * height (cm) – 5 * age (y) + s (kcal / day)
```

Comment:
Comments should not be pushed to the main

Reply:
This comment is to explain the formula and where the numbers come from, to prevent magic numbers.
Without it, it might be very confusing if someone who doesn't know the Mifflin St Jeor equation looks at the code